README.md

```diff
@@ -1,3 +1,59 @@
 # Data Models
 
 Data models and vocabularies in the biomedical space.
+
+## Persistent CSV Format
+
+Data model descriptions are stored persistently in this repository in csv format for portability and human readability. Each data model has its own directory with versions of the model in subdirectories. Each version has at least `definitions` and `schema` directories and, optionally, a `constraints` directory and `indexes.csv` and `references.csv` files.
```

Comment:
Capitalize CSV (in the whole document). I will describe a couple of implementation details:

- The `datamodel.json` file is what *signals* that a data model exists in the current directory a subdirectory
- From the service's standpoint, directories don't actually matter. Files are detected based on *signature* of the CSV header. That being said, it is still good to describe a convention to follow so contributors don't need to make the decisions themselves and each repository becomes familiar.